sdk/custom/intents/READ_GOTO_LIBRARY.yaml

```diff
@@ -0,0 +1,11 @@
+trainingPhrases:
```

Comment:
nit: change the file name to lowercase so that it is uniform with the rest of the project